evm/gas-service.js

```diff
@@ -86,23 +100,26 @@ async function getGasUpdates(config, env, chain, destinationChains) {
             }
 
             const {
-                source_base_fee: sourceBaseFee,
-                source_express_fee: { total: sourceExpressFee },
-                source_token: {
-                    gas_price_in_units: { value: gasPrice },
+                destination_native_token: {
                     token_price: { usd: srcTokenPrice },
-                    decimals,
                 },
+            } = sourceFeeData;
+
+            const {
+                base_fee_usd: baseFeeUsd,
+                destination_express_fee: { total_usd: expressFeeUsd },
                 destination_native_token: {
-                    token_price: { usd: destinationTokenPrice },
+                    token_price: { usd: destTokenPrice },
+                    gas_price_in_units: { value: gasPriceInDestToken },
                 },
                 execute_gas_multiplier: multiplier = 1.1,
-            } = data.result;
+            } = destinationFeeData;
 
-            const axelarBaseFee = parseFloat(sourceBaseFee) * Math.pow(10, decimals);
-            const expressFee = parseFloat(sourceExpressFee) * Math.pow(10, decimals);
-            const relativeGasPrice = parseFloat(gasPrice) * parseFloat(multiplier) * parseFloat(estimateMultiplier);
-            const gasPriceRatio = parseFloat(destinationTokenPrice) / parseFloat(srcTokenPrice);
+            const axelarBaseFee = parseFloat(baseFeeUsd) / parseFloat(srcTokenPrice);
```

Comment:
This value will be in terms of the source token (e.g. 10 FTM etc.). We still need to convert it to wei

Same below